editor/src/messages/input_mapper/utility_types/input_keyboard.rs

```diff
@@ -280,36 +314,31 @@ pub const NUMBER_OF_KEYS: usize = Key::NumKeys as usize;
 #[derive(Debug, Clone, Default, PartialEq, Eq, Serialize, Deserialize)]
 pub struct KeysGroup(pub Vec<Key>);
 
-impl KeysGroup {
-	pub fn keys_text_shortcut(&self, keyboard_platform: KeyboardPlatformLayout) -> String {
-		const JOINER_MARK: &str = "+";
+impl fmt::Display for KeysGroup {
+	fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+		const JOINER_MARK: &str = " ";
 
 		let mut joined = self
 			.0
 			.iter()
 			.map(|key| {
-				let key_string = key.to_string();
-
-				if keyboard_platform == KeyboardPlatformLayout::Mac {
-					match key_string.as_str() {
-						"Command" => "⌘".to_string(),
-						"Control" => "⌃".to_string(),
-						"Alt" => "⌥".to_string(),
-						"Shift" => "⇧".to_string(),
-						_ => key_string + JOINER_MARK,
-					}
+				let keyboard_layout = GLOBAL_PLATFORM.get().expect("Failed to get GLOBAL_PLATFORM").as_keyboard_platform_layout();
+				let key_is_modifier = *key == Key::Control || *key == Key::Command || *key == Key::Alt || *key == Key::Shift || *key == Key::Meta || *key == Key::Accel;
```

Comment:
I think this can be shortend

Reply:
Nice, I'll try and remember this in the future, thanks.